pysnooper/tracer.py

```diff
@@ -19,10 +19,14 @@
 
 
 def get_local_reprs(frame, watch=()):
-    result = {key: utils.get_shortish_repr(value) for key, value
-                                                     in frame.f_locals.items()}
+    vars_order = frame.f_code.co_varnames + frame.f_code.co_cellvars + frame.f_code.co_freevars + tuple(frame.f_locals.keys())
```

Comment:
OK, I thought that co_cellvars must always be a subset of co_varnames, but I see that it's not. Which is really weird.

```suggestion
    code = frame.f_code
    vars_order = code.co_varnames + code.co_cellvars + code.co_freevars + tuple(frame.f_locals.keys())
```